code/python/create_template_from_Syn_schema.py

```diff
@@ -0,0 +1,270 @@
+#!/usr/bin/env python3
+
+"""
+Program: create_template_from_Syn_schema.py
+
+Purpose: Use a JSON schema registered in Synapse to generate either csv file
+         templates or an Excel workbook containing template worksheets.  The
+         following will be generated as either separate files (csv) or
+         worksheets within the workbook (Excel):
+         - a blank template to use for data entry
+         - a dictionary defining the columns in the template
+         - a listing of allowable values for columns that use a controlled
+           vocabulary list
+
+Input parameters: Full name of the registered schema, including
+                      organization
+                  Full pathname to the output template file
+                  Desired output - either csv or excel
+
+Outputs: csv template file or Excel workbook
```

Comment:
```suggestion
Outputs: csv template files or Excel workbook
```
Since it will be three files.